src/data-interfaces/HoloFuelDnaInterface.js

```diff
@@ -295,8 +337,8 @@ const HoloFuelDnaInterface = {
       return presentedNonActionableTransactions.sort((a, b) => a.timestamp > b.timestamp ? -1 : 1)
     },
     getPending: async (transactionId) => {
-      const { requests, promises } = await createZomeCall('transactions/list_pending')({ origins: transactionId })
-      const transactions = requests.map(r => presentPendingRequest(r)).concat(promises.map(p => presentPendingOffer(p)))
+      const { requests, promises } = await getListPending({ origins: transactionId })
+      const transactions = await requests.map(r => presentPendingRequest(r)).concat(promises.map(p => presentPendingOffer(p[0], p[1]))).filter(tx => !(tx instanceof Error))
```

Comment:
seems like we don't need this await

Reply:
Good catch; this is definitely residue from prior versions of the code.